README.md

```diff
@@ -25,6 +25,68 @@ event scraper functions.
 
 ## Quick Start
 
+### Scrapers
+
+#### Event scraper structure
+
+(show get_events.png)
+<!-- ![get_events](./docs/_static/get_events.png) -->
```

Comment:
Yea this is the stupid part of image / asset management.

The way I have done this in the past is make a directory in the top level dir of the repo called `images` and all paths then become `./images/{file}`

Then in the `make gen-docs` command I copy the images over to the newly generated `docs/html` dir: https://github.com/CouncilDataProject/cdp-scrapers/blob/main/Makefile#L51

You can see this behavior in some prior work of mine here.
README with image: https://github.com/AllenCellModeling/actk
Makefile: https://github.com/AllenCellModeling/actk/blob/master/Makefile#L51